### PR TITLE
Update the "Keeping Ballerina Upto Date" Learn Page for Swan Lake Beta

### PR DESCRIPTION
**Pull request by parkavi11**
## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by praneesha — APPROVED** [review]

**Review by keizer619 — APPROVED** [review]

File: learn/tooling-guide/cli-tools/update-tool.md [hunk 1]
```diff
@@ -30,28 +30,6 @@ Ballerina distributions are released via two different release channels at the m
 
 Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
 
-#### Versioning of the releases
```
Comment (keizer619) [inline learn/tooling-guide/cli-tools/update-tool.md:33]: Its ok to remove this as now we don't follow semver versions. 
Shall we mention that swan lake release channel is the primary release and mention that 1.2.x release will be done on maintenance mode with patches?  
@parkavi11 @praneesha

**Review by keizer619 — COMMENTED** [review]
Shall we mention that swan lake release channel is the primary release and mention that 1.2.x release will be done on maintenance mode with patches?
@parkavi11 @praneesha

File: learn/tooling-guide/cli-tools/update-tool.md (hunk 1 above)
Comment (praneesha) [reply learn/tooling-guide/cli-tools/update-tool.md:33]: +1

File: learn/tooling-guide/cli-tools/update-tool.md [hunk 2]
```diff
@@ -26,32 +26,16 @@ Ballerina distribution is a term, which refers to these jBallerina and nBallerin
 
 ### Getting to know the Release Channels
 
-Ballerina distributions are released via two different release channels at the moment. One is the Swan Lake release channel and the other one is the 1.x release channel.
+Ballerina distributions are released via two different release channels at the moment. One is the Swan Lake release channel and the other one is the 1.x release channel. Swan lake release channel is the primary release and 1.2.x release will be done on maintenance mode.
```
Comment (keizer619) [inline learn/tooling-guide/cli-tools/update-tool.md:29]:
```suggestion
Ballerina distributions are released via two different release channels at the moment. One is the Swan Lake release channel and the other one is the 1.x release channel. Swan lake release channel is the primary release and 1.x releases will be done on maintenance mode.
```

File: learn/tooling-guide/cli-tools/update-tool.md (hunk 1 above)
Comment (hevayo) [reply learn/tooling-guide/cli-tools/update-tool.md:33]: we should not mention that 1.2.x is on maintenance mode specifically

**Review by hevayo — APPROVED** [review]

**Pull request merged**